source/_posts/2017-12-08-UIGestures.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  UIGestures
+date: 2017-12-08 13:00:00
+tags: ios,accelerators,gestures
+authorIds:
+- roju
+categories:
+- iOS
+---
+
+Gestures are the magic behind those accelerators that can make your app more user friendly. We will show now a few examples on how to implement them in your code.
+
+## [Swipe Gestures](https://developer.apple.com/documentation/uikit/uiswipegesturerecognizer)
+
+You can have many accelerators build with the swipe gesture as it is discrete and you can have many combinations as you can select the direction of the swipe and the number of touches required. Even if we have the possibility to set up a high number of touches, a maximum of 3 is still user friendly. You should also make sure you keep some sort of consistency when defining the gestures, if swipe right with 2 fingers goes to the previous conversation in a messaging app, swiping left with 2 fingers should probably go to the next conversation.
+
+### Using the storyboard
+
+{% vimeo 244338896 %}
```

Comment:
Is there someway to make these videos display larger?
![screen shot 2017-12-18 at 14 37 52](https://user-images.githubusercontent.com/20965430/34109155-01ab8b10-e403-11e7-8c15-0b0b2c59bb79.png)